packages/amplify-category-storage/src/provider-utils/awscloudformation/service-walkthrough-types/s3-user-input-types.ts

```diff
@@ -0,0 +1,29 @@
+//user input-data types generated by cli
+
+export function enumToHelp( obj : object ){
+    return `One of ${Object.values(obj)}`;
+}
+
+export enum S3AccessType  {
+    AUTH_AND_GUEST =  "authAndGuest",
+    AUTH_ONLY =  "auth"
+}
+
+export enum S3PermissionType {
```

Comment:
I would change this to something human readable as what we have in the CLI - Create/Read/Update/Delete.